src/pages/academia.js

```diff
@@ -86,12 +162,106 @@ export default () => {
                         </StyledButton>
                     </HeaderContent>
                 </Header>
-            </HeroBanner>
+            </StyledHeroBanner>
 
             <BodyContent>
-                <div ref={formRef}>
-                    <AcademiaSignUpForm />
+                <div>
```

Comment:
So the design does a small overlap between the image and the text like so:

![Screen Shot 2020-05-27 at 9 58 35 AM](https://user-images.githubusercontent.com/9064401/83029499-baa45f00-a000-11ea-808f-cd077511c5d6.png)

You are definitely right sticking to the 780px max width, Dan's design looks to be like 850 which gives the image more space. For implementation though, let's stick to the 780px max width and shrink this image to 450 or 500px, I'll leave that up to you. That way we can give more room to the text since it was pretty narrow before:

![Screen Shot 2020-05-27 at 10 03 31 AM](https://user-images.githubusercontent.com/9064401/83030011-5766fc80-a001-11ea-97c6-c6ee9246eef7.png)